submissions/damilolaSalami/html/productpage.html

```diff
@@ -11,10 +11,10 @@
         <h1><a href="#">BuyMe</a></h1>
         <nav>
             <ul>
-                <li><a href="#">LogIn</a></li>
+                <li><a href="#">Log In</a></li>
                 <li><a href="#">Create a profile</a></li>
-                <li><a href="#">Cart</a></li>
-                <li><a href="#">Contact us</a></li>
+                <li><a href="../html/productpage.html"></a>Product page</li>
```

Comment:
simply do `href="productpage.html"`. Also, what about your contact form page? Are you not linking pages on that too?